e2e_deployment_files/edgehub_restart_deployment.template.json

```diff
@@ -0,0 +1,230 @@
+              },
+              "testDuration": {
+                "value": "<TestDuration>"
+              },
+              "testStartDelay": {
+                "value": "<TestStartDelay>"
+              },
+              "trackingId": {
+                "value": "<TrackingId>"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-edgehub-restart-tester:<Build.BuildNumber>-linux-<Architecture>",
+              "createOptions": "{\"HostConfig\":{\"NetworkMode\":\"host\"},\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}}}"
+            }
+          },
+          "relayer1": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "transportType": {
+                "value": "Amqp"
+              },
+              "inputName": {
+                "value": "input1"
+              },
+              "outputName": {
+                "value": "output1"
+              },
+              "testResultCoordinatorUrl": {
+                "value": "http://testResultCoordinator:5001"
+              }
+            },
+            "settings": {
+              "image": "<Container_Registry>/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
+              "createOptions": ""
+            }
+          },
+          "testResultCoordinator": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "Logging:LogLevel:Microsoft": {
+                "value": "Error"
+              },
+              "trackingId": {
+                "value": "<TrackingId>"
+              },
+              "testStartDelay": {
+                "value": "<TestStartDelay>"
+              },
+              "testDuration": {
+                "value": "<TestDuration>"
+              },
+              "verificationDelay": {
+                "value": "<TestResultCoordinator.VerificationDelay>"
+              },
+              "eventHubConnectionString": {
+                "value": "<TestResultCoordinator.EventHubConnectionString>"
+              },
+              "ConsumerGroupName": {
+                "value": "<TestResultCoordinator.ConsumerGroupId>"
+              },
+              "optimizeForPerformance": {
+                "value": "<TestResultCoordinator.OptimizeForPerformance>"
+              },
+              "logAnalyticsWorkspaceId": {
+                "value": "<LogAnalyticsWorkspaceId>"
+              },
+              "logAnalyticsSharedKey": {
+                "value": "<LogAnalyticsSharedKey>"
+              },
+              "logAnalyticsLogType": {
+                "value": "<TestResultCoordinator.LogAnalyticsLogType>"
+              },
+              "IOT_HUB_CONNECTION_STRING": {
+                "value": "<IoTHubConnectionString>"
```

Comment:
copy "STORAGE_ACCOUNT_CONNECTION_STRING" environment variable from master copy.

Reply:
Deferred this comment to https://msazure.visualstudio.com/One/_workitems/edit/6186177

Reply:
You need to have it in this PR as I think connectivityTest.sh requires this parameter.